pkg/commands/capability.go

```diff
@@ -22,12 +22,20 @@ func CapabilityCommandGroup(c types.Args, ioStream cmdutil.IOStreams) *cobra.Com
 		Use:   "cap",
 		Short: "Manage capability centers and installing/uninstalling capabilities",
 		Long:  "Manage capability centers and installing/uninstalling capabilities",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			restConf, err := config.GetConfig()
+			if err != nil {
+				return err
+			}
+			c.Config = restConf
+			return nil
+		},
```

Comment:
I  see lot's of places has the same function, do you mind to write a common one?

Reply:
@wonderflow Thanks for your comments, I create a common function in https://github.com/oam-dev/kubevela/blob/5e4f270b1e50e16b0a670ee6b3ac4d94bbc90e86/apis/types/args.go#L16-L23